code/socialDistribution/templatetags/post.py

```diff
@@ -44,7 +44,9 @@ def card_post(post, author):
         'content_media': content_media, 
         'isAuthor': isAuthor, 
         'isLiked': isLiked, 
-        'likeText': likeText
+        'likeText': likeText,
+        'editPostModalId': 'edit_post_modal' + str(post.id),
+        'deletePostModalId': 'delete_post_modal'+ str(post.id)
```

Comment:
Is it necessary to send the post.id twice? It just seems a bit redundant. Could we pass in a single `postid` attribute and use string concatenation in `post.html` to generate the edit modal id and delete modal id?

Reply:
sure, thanks